README.md

```diff
@@ -14,17 +14,71 @@ The TSC for Presto Foundation can be reached at their [mailing list](https://lis
 
 Once you join the The [Presto Foundation Slack Workspace](https://join.slack.com/t/prestodb/shared_invite/enQtNTQ3NjU2MTYyNDA2LTYyOTg3MzUyMWE1YTI3Njc5YjgxZjNiYTgxODAzYjI5YWMwYWE0MTZjYWFhNGMwNjczYjI3N2JhM2ExMGJlMWM), you can participate in any public channels.
 
 ### Calendars and Meetings
 
 The Presto Foundation maintains a [public calendar](https://calendar.google.com/calendar/embed?src=linuxfoundation.org_vrjlva5b0u73ps75fvnv5sasi4%40group.calendar.google.com&ctz=America%2FChicago) for TSC meetings. These meetings are open for anyone to join.
 
+Because we work in a highly distributed environment and will rarely meet in person, participants are encouraged to use video as appropriate.
 
 ## Members
 
-The TSC voting members are initially the [Project’s Committers](https://github.com/prestodb/presto/wiki/committers), and the current TSC chair is Nezih Yigitbasi ([@nezihyigitbasi](https://github.com/nezihyigitbasi)). At the inception of the project, the Committers of the Project will be as set forth within the "CONTRIBUTING" file within the Project’s code repository
+The TSC voting members are initially the [Project’s Committers](https://github.com/prestodb/.github/blob/master/CONTRIBUTING.md).
 
-https://github.com/prestodb/presto/blob/master/CONTRIBUTING.md
+The current members of the Presto TSC are:
+
+| Name                      | Github                                              | Affiliation | Term begins  | Term ends    |
+| ------------------------- | --------------------------------------------------- | ------------|--------------|--------------|
+| Nezih Yigitbasi (chair)   |[@nezihyigitbasi](https://github.com/nezihyigitbasi) | Facebook    | 1 Jan, 2020  | TBD          |
+| Masha Basmanova           |[mbasmanova](https://github.com/mbasmanova)          | Facebook    | 1 Jan, 2020  | TBD          |
+| Zhenxiao Luo              |[zhenxiao](https://github.com/zhenxiao)              | Twitter     | 1 Jan, 2020  | TBD          |
+| Andrii Rosa               |[arhimondr](https://github.com/arhimondr)            | Facebook    | 1 Jan, 2020  | TBD          |
+| Rebecca Schlussel         |[rschlussel](https://.github.com/rschlussel)         | Facebook    | 1 Jan, 2020  | TBD          |
+| James Sun                 |[highker](https://github.com/highker)                | Facebook    | 1 Jan, 2020  | TBD          |
+| Wenlei Xie                |[wenleix](https://github.com/wenleix)                | Facebook    | 1 Jan, 2020  | TBD          |
+| Rongrong Zhong            |[rongrong](https://github.com/rongrong)              | Facebook    | 1 Jan, 2020  | TBD          |
+| Shixuan Fan               |[shixuan-fan](https://github.com/shixuan-fan)        | Facebook    | 1 Jan, 2020  | TBD          |
+| Jiexi Lin                 |[jessesleeping](https://github.com/jessesleeping)    | Facebook    | 1 Jan, 2020  | TBD          |
```

Comment:
Remove?

Reply:
I think theoretically we should not remove @jessesleeping unless he resigned or some rules would remove him from the list? Not sure about how it works.